templates/ovndbcluster/bin/setup.sh

```diff
@@ -129,4 +132,6 @@ if [[ "$(hostname)" == "{{ .SERVICE_NAME }}-0" ]]; then
     unset OVN_${DB_TYPE^^}_DAEMON
 fi
 
+trap - EXIT
```

Comment:
`run_${DB_TYPE}_ovsdb` above is started in background (`&`). So we don't really know here whether the tool is done, yet. So it's probably premature to remove the trap. Should we `wait_for_ovsdb_tool` here before removing the trap?

Reply:
I've spent several days trying to make the kuttl tests work with this proposed change with no luck. Let me push a new PS to show you what is happening.

Reply:
Done